### DISCUSSION TITLE
Can you add a dual problem solution?

### DISCUSSION THREAD

**Discussion by Keon6** [qpsolvers/qpsolvers#118] [Ideas]
Could you add the solution to the dual problem as well?

**Comment by stephane-caron** [comment]
On it :+1: Issue: https://github.com/stephane-caron/qpsolvers/issues/124, PR: https://github.com/stephane-caron/qpsolvers/pull/123.

**Reply by stephane-caron** [reply]
And the feature is now available with v2.7.0 :smiley: Check out *e.g.* the [release notes](https://github.com/stephane-caron/qpsolvers/releases/tag/v2.7.0) for an example on how to use it.
